index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Protofolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;800&display=swap" rel="stylesheet">
+</head>
+<body>
+    
+    <section class="toolbar">
+        <nav class="flex-nav">
+            <a href="#" class="toggle-nav"> <img src="images/Icon - Menu.png" alt=""></menu></a>
+            <ul>
+                <li><a href="#"> Protofolio</li></a>
+                <li><a href="#"> About</li></a>
+                <li><a href="#"> Contact</li></a>
+
+               
+
+            </ul>
+        </nav>
+
+    </section>
+
+    <section class="headline">
+        <h1 class="primary">
+            Hey there. I’m Nessrine.
+            <br>
+            <span>I’m a software developer</span> 
+        </h1>
+        <p class="supporting-text">
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+        </p>
+        <div class="social-media">
+            <img src="images/social-media/github.svg" alt="github-icon">
+            <img src="images/social-media/Linkedin icon.svg" alt="linkedin-icon">
+            <img src="images/social-media/angellist.svg" alt="angelist-icon">
+            <img src="images/social-media/twitter.svg" alt="twitter-icon">
+            <img src="images/social-media/medium.svg" alt="medium-icon">
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href. Kindly address that too.